### PR TITLE
Overhaul CONTRIBUTING.rst

### PR DESCRIPTION
**Pull request by Kodiologist**
Since this PR involves policy changes, it should have two reviewers.

### CODE REVIEW

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -1,83 +1,125 @@
...
-helps in making Hy more awesome.
+Contributions are welcome and greatly appreciated. Every little bit
+helps in making Hy better. Potential contributions include:
 
-Pull requests are great! We love them; here is a quick guide:
+- Reporting bugs
+- Fixing bugs
+- Requesting features
+- Adding features
+- Writing tests for outstanding bugs or untested features
```
Comment (tuturto) [inline CONTRIBUTING.rst]: I would explicitly mention writing documentation here

**Comment by Kodiologist** [comment]
@kirbyfan64 @gilch @tuturto I can has review?

**Comment by kirbyfan64** [comment]
:+1:! I made a couple of language and grammar changes/fixes, so @Kodiologist please look them over before this is merged to make sure there's nothing that you disagree with.

**Comment by Kodiologist** [comment]
Thanks, Ryan.

**Pull request merged**